includes/features/SU-SWS/stanford_page/stanford_page_3.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Stanford Paragraph Types
+  In order to ensure that the Stanford Paragraphs Types installs properly
+  As an end user
+  I want to verify the display of content.
+
+  Background:
```

Comment:
You are not allowed to tag a background (why? *shrug*) But you CAN tag a whole featue. If your background requires administrator role you could put the @api tag as the very first line of this document. That way all the scenarios get it without having to remember to put it there for each one.